tar_scm.py

```diff
@@ -54,6 +54,10 @@ def run_cmd(cmd, cwd, interactive=False, raisesysexit=False):
     env = os.environ.copy()
     env['LANG'] = 'C'
 
+    if is_proxy_defined():
+        env['http_proxy'] = os.environ.get('http_proxy')
```

Comment:
This variable comes in lower-case from our `/etc/profile.d/profile.sh` and is also documented as lowercase in `man curl` so should stay this way.